index.d.ts

```diff
@@ -11,16 +11,57 @@ export declare class ActionsObservable<T extends Action> extends Observable<T> {
    * which isn't possible in either JavaScript or TypeScript. So instead, we
    * provide safe typing for up to 6 items, following by a scheduler.
    */
-  static of<T extends Action>(item1: T, scheduler?: Scheduler): ActionsObservable<T>;
-  static of<T extends Action>(item1: T, item2: T, scheduler?: Scheduler): ActionsObservable<T>;
-  static of<T extends Action>(item1: T, item2: T, item3: T, scheduler?: Scheduler): ActionsObservable<T>;
-  static of<T extends Action>(item1: T, item2: T, item3: T, item4: T, scheduler?: Scheduler): ActionsObservable<T>;
-  static of<T extends Action>(item1: T, item2: T, item3: T, item4: T, item5: T, scheduler?: Scheduler): ActionsObservable<T>;
-  static of<T extends Action>(item1: T, item2: T, item3: T, item4: T, item5: T, item6: T, scheduler?: Scheduler): ActionsObservable<T>;
-  static of<T extends Action>(...array: Array<T | Scheduler>): ActionsObservable<T>;
+  static of<T extends Action>(
```

Comment:
ehhh 👎 but I totally get it. It's one of those "in this specific case I disagree, but in most cases I would agree"

Reply:
yeah, adding prettierignore block can be good here